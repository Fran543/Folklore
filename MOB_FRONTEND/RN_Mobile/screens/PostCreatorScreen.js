import { Text, View } from "react-native";

export default function PostCreatorScreen(){
    return(
        <View>
            <Text>Post creator!</Text>
        </View>
    );
}