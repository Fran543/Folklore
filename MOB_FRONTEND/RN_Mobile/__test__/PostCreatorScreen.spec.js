// import {default as HomeScreeen} from "../screens/HomeScreen"

// describe('truth', () => {
//     it('is true', () => {
//       expect(true).toEqual(true);
//     });
//   });

import React from 'react';
import renderer from 'react-test-renderer';

import PostCreatorScreen from '../screens/PostCreatorScreen';

describe('<PostCreatorScreen />', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<PostCreatorScreen />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('has 2 child', () => {
        const tree = renderer.create(<PostCreatorScreen />).toJSON();
        expect(tree.children.length).toBe(2);
    });
});
