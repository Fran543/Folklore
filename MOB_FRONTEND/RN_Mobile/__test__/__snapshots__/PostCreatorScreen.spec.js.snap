// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostCreatorScreen /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "black",
      "flex": 1,
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "blue",
        "borderRadius": 20,
        "flex": 1,
        "justifyContent": "center",
        "marginVertical": 20,
        "width": "80%",
      }
    }
  >
    <View>
      <Text
        style={
          Object {
            "color": "white",
            "fontWeight": "bold",
            "letterSpacing": 3,
            "textTransform": "uppercase",
          }
        }
      >
        Blog
      </Text>
    </View>
  </View>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#f5bc1d",
        "borderRadius": 20,
        "flex": 1,
        "justifyContent": "center",
        "marginVertical": 20,
        "width": "80%",
      }
    }
  >
    <View>
      <Text
        style={
          Object {
            "color": "white",
            "fontWeight": "bold",
            "letterSpacing": 3,
            "textTransform": "uppercase",
          }
        }
      >
        Interactive Story
      </Text>
    </View>
  </View>
</View>
`;
